usaspending_api/etl/management/commands/load_transactions_in_delta.py

```diff
@@ -1094,16 +1114,14 @@ def initial_run(self, next_last_load):
                 INSERT OVERWRITE {destination_database}.{destination_table}
                     SELECT
                         existing_awards.id AS award_id,
-                        existing_awards.detached_award_procurement_id,
```

Comment:
Comment above
> \# Insert existing transactions into the lookup table

Should probably say "awards" in instead of "transactions"

Reply:
Actually, this table maps individual transactions to their corresponding award_id, so the comment is correct.

Reply:
I decided to update the comment to make this clearer.